SPECS/libuser/libuser.spec

```diff
@@ -149,6 +146,9 @@ export PYTHONPATH
 %endif
 
 %changelog
+* Wed Aug 28 2024 Reuben Olinsky <reubeno@microsoft.com> - 0.64-1
```

Comment:
Couldn't add the comment in the `%check` section, adding it here:

The way it's detected if a ptest have failed or succeeded in our builds, is from the exit code of the last command. Not ideal, and there's some history behind that, I can get into details offline, if you're curious. Bottom line right now is that the way `%check` is currently written, a failing `%make_build check` run will not mark the tests as failing. Considering that, I'd like to ask you to reorganize `%check` a little.

One way we've worked around this elsewhere is by keeping a `test_ok` variable initialized to `true` and tracking return codes for all ran test commands and running `$test_ok` as the last command. Every time we see an error, we'd do `test_ok=false`. That's not required, though, it's just one idea. Example in `edk2.spec`.

Reply:
Yes, would be great to learn more about that history sometime; but thanks for the context regardless. I've taken your suggestion and applied it.